src/main/java/com/olgaboiar/mint/Request.java

```diff
@@ -1,31 +1,22 @@
 package com.olgaboiar.mint;
 
+import java.net.URL;
+
 public class Request {
     private String method;
-    private String uri;
-    private String protocol;
-
-    public String getMethod() {
-        return method;
-    }
-
-    public String getUri() {
-        return uri;
-    }
+    private URL url;
 
-    public String getProtocol() {
-        return protocol;
-    }
-
-    void setMethod(String method) {
+    public Request(URL url, String method) {
+        this.url = url;
         this.method = method;
-    }
 
-    void setRequestedFile(String uri) {
-        this.uri = uri;
     }
 
-    void setProtocol(String protocol) {
-        this.protocol = protocol;
+    public String getMethod() {
+        return method;
+    }
+//
```

Comment:
⛔️

Reply:
removed in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/4713bbd8fc081a7764a438b7c5a2c62956bad05f